server/game/cards/03_TWI/units/WolfPackEscort.ts

```diff
@@ -0,0 +1,27 @@
+import AbilityHelper from '../../../AbilityHelper';
+import { NonLeaderUnitCard } from '../../../core/card/NonLeaderUnitCard';
+import { CardType, RelativePlayer, Trait } from '../../../core/Constants';
+
+export default class WolfPackEscort extends NonLeaderUnitCard {
+    protected override getImplementationId () {
+        return {
+            id: '1882027961',
+            internalName: 'wolf-pack-escort'
+        };
+    }
+
+    public override setupCardAbilities() {
+        this.addWhenPlayedAbility({
+            title: 'You may return a friendly non-leader, non-Vehicle unit to its owner\'s hand.',
+            optional: true,
+            targetResolver: {
+                cardTypeFilter: CardType.BasicUnit,
```

Comment:
Should be `WildcardCardType.NonLeaderUnit`. `BasicUnit` excludes tokens, it is just "units" and not "leader units" or "token units"